src/steps/trackTransaction.ts

```diff
@@ -157,13 +159,18 @@ export function trackTransaction(
     tr.getMeta('inputType') && newTr.setMeta('inputType', tr.getMeta('inputType'))
     tr.getMeta('uiEvent') && newTr.setMeta('uiEvent', tr.getMeta('uiEvent'))
   }
+  if (setsNewSelection && tr.selection instanceof TextSelection) {
+    // preserving text selection if we track an element in which selection is set
+    const newPos = newTr.doc.resolve(tr.selection.from) // no mapping on purpose as tracking will misguide mapping
+    newTr.setSelection(new TextSelection(newPos))
+  }
   // This is kinda hacky solution at the moment to maintain NodeSelections over transactions
   // These are required by at least cross-references and links to activate their selector pop-ups
   if (wasNodeSelection) {
     console.log('%c Getting into node select! ', 'background: #222; color: #bada55')
     // And -1 here is necessary to keep the selection pointing at the start of the node
```

Comment:
Have you checked the comments on L171 and L172? Do you want to remove them if they're no longer relevant?

Reply:
I restored that, this wasn't supposed to be changed